src/_guides/language/effective-dart/design.md

```diff
@@ -532,7 +532,7 @@ doesn't know if any code outside of its view is using it.
 
 ### CONSIDER declaring multiple classes in the same library.
 
-Java and C# tie the organization of files to the organization of
+Java ties the organization of files to the organization of
```

Comment:
Maybe we shouldn't tie this to Java so strongly (unless Java really is the only language that does this). Perhaps:

Some languages, such as Java, tie the organization...